priv/templates/phx.gen.auth/auth_helpers.ex

```diff
@@ -0,0 +1,31 @@
+defmodule <%= inspect context.web_module %>.<%= if schema.web_path, do: "#{Phoenix.Naming.humanize(schema.web_path)}.", else: "" %><%= Phoenix.Naming.humanize(schema.singular) %>AuthHelpers do
+  import Phoenix.LiveView, only: [assign_new: 3, connected?: 1]
+
+  @doc """
+  Attaches current_<%= schema.singular %> to `socket` assigns based on <%= schema.singular %>_token.
+
+  ## Example
+      # In the LiveView file
+      import <%= inspect context.web_module %>.<%= if schema.web_path, do: "#{Phoenix.Naming.humanize(schema.web_path)}.", else: "" %><%= Phoenix.Naming.humanize(schema.singular) %>AuthHelpers
+
+      # LiveView mount
+      def mount(_params, session, socket) do
+        socket = mount_current_<%= schema.singular %>(socket, session)
+        {:ok, socket}
+      end
+
+  """
+  def mount_current_<%= schema.singular %>(socket, %{"<%=schema.singular %>_token" => <%= schema.singular %>_token}) do
+    if connected?(socket) do
```

Comment:
Why the `connected?` branch? :)

Reply:
At first it was to prevent duplicating the call to get_user_by_session_token/1, but assign_new/3 can do the job alone :)